### DISCUSSION TITLE
空洞内疑似出现超大型以骸

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2475] [其他]
![image](https://tse1-mm.cn.bing.net/th/id/OIP-C.uFNigZ47R_H0VTjpZO969wHaEM?rs=1&pid=ImgDetMain)
今天在空洞做委托的时候找了个安全的地方休息 休息的时候抬头一看就看到了这么大的玩意 给我吓个半死
就在我准备直接跑路离开空洞的时候这大东西直接消失了 还好我每次下空洞都会开着录制 留下了图像
这是什么新种类以骸吗 看着就可怕

**Comment by Aoer2024** [comment]
这个委托我接不了

**Comment by kiana127127127** [comment]
全网呼叫aa人。
